docker-compose.yml

```diff
@@ -1,37 +1,23 @@
-version: '3.7'
-
 services:
   db:
     image: mariadb:10.2
     restart: unless-stopped
     volumes:
       - mariadb:/var/lib/mysql
-    extends:
-      service: base
-      file: docker-compose-base.yml
+    env_file: '.env'
 
   rmt:
     build: .
     restart: unless-stopped
-    environment:
-      - MYSQL_HOST=db
-      - SECRET_KEY_BASE=${SECRET_KEY_BASE}
-      - SCC_USERNAME=${SCC_USERNAME}
-      - SCC_PASSWORD=${SCC_PASSWORD}
+    env_file: '.env'
     volumes:
       - .:/srv/www/rmt
-      - public:/srv/www/repo/public/
+      - ./config/rmt.yml:/etc/rmt.conf
```

Comment:
What is the case for removing the `public` volume? I see it as necessary if we want to have consistency between the database and the disk for downloaded files.

Reply:
I thought its easier to comprehend wehn it takes it from your local `public/` directory instead of having a separate volume. This way it is not hidden in a volume and you can access it from outside.

Do you see a downside to this?

Reply:
Maybe I'm confusing things here, but this volume should deny you access from outside the container, it will keep downloaded files on your local `public/`, but actually mounted as another volume on the Docker container.

I'm uncertain about the motivation for the volume, but I'd guess it's to keep the environment closer to an actual RMT installation, as I suppose `/srv/www/repo/public/` would be the path used on production systems.

Reply:
After revisting this PR: The directory `/srv/www/rmt` is mounted to `.` this also includes the `public` directory from the local directory. Do you think the extra `volume` brings any extra value here? Maybe that its easier to transfer?

Reply:
We talked about this in our pairing session. `/srv/www/rmt/public` is mounted by `- .:/srv/www/rmt`. I do not see the point why this needs to be an extra volume. Mounting the `public` directory from the host system to somewhere different works fine.